pkg/apis/componentconfig/cloudcore/v1alpha1/default.go

```diff
@@ -172,7 +172,7 @@ func NewDefaultCloudCoreConfig() *CloudCoreConfig {
 				RestTimeout: 60,
 			},
 			MissionStatePruner: &MissionStatePruner{
-				Enable:             false,
+				Enable:             true,
```

Comment:
what would be the case for this Enable to be false?

Reply:
This component is added by us, so it is unnecessary if the edge-core is not running in edge-cluster mode.

Reply:
I see. thanks!